microsoft-edge/origin-trials/index.md

```diff
@@ -1,26 +1,172 @@
+
+1. Select an active origin trial.
+
+1. Click the **Issues** button next to the registration button for the origin trial.
+
+
+<!-- ====================================================================== -->
+## Renew an origin trial token
+
+Origin trial tokens expire every 6 weeks by default.  After the origin trial token expires, you must renew the token.  You can renew the token before it expires.
+
+To renew an origin trial token that has expired or is about to expire:
+
+1. Go to **My Registered Trials** and select an origin trial.
+
+1. Click **Renew**.  A fresh token is sent to you.
+
+1. Copy the new token and use it on your site; see [How to use an origin trial token](#how-to-use-an-origin-trial-token), above.
+
+
+<!-- ====================================================================== -->
+## Developer guidelines
+
+
+<!-- ------------------------------ -->
+#### Allowed origins
+
+Edge origin trials only support SSL-enabled domains. For localhost testing of a feature, please use the appropriate Edge Feature Flag located in `edge://flags`. In the future, we may publish public tokens that are valid for localhost testing. 
+
+
+<!-- ------------------------------ -->
+#### Subdomains, URL paths, and query parameters
+
+
+<!-- ---------- -->
+###### Subdomains
+
+Origin trial tokens can be configured to support any of the following:
+* A single domain, without supporting its subdomains.  For example, register `https://example.com`, and in the **Subdomain Support** section, select the **No** option button.
+* A single subdomain.  For example, register `https://beta.example.com`, and in the **Subdomain Support** section, select the **No** option button.
+* A domain and its subdomains.  For example, register `https://example.com`, and in the **Subdomain Support** section, select the **Yes** option button.
+<!-- todo: check section heading & option button labels at microsoft.com portal; the above ui is from github.io portal -->
+
+
+<!-- ---------- -->
+###### Paths in URLs, and query parameters
+
+In URLs, paths and query parameters aren't supported by the origin trial system.  When registering for an origin trial, if a URI is submitted that has a trailing path or a query parameter, such as `https://example.com/path/new-feature`, the root domain (`https://example.com`) will be registered, without the trailing path or query parameter.
+
+
+<!-- ------------------------------ -->
+#### Duration of an origin trial
+
+
+<!-- ---------- -->
+###### A scheduled expiration date
+
+Once you have registered for a trial, your token will be valid until the expiration date that's listed on the origin trial's informational card.  After this date, the token will no longer function, and the feature that's associated with the trial will be considered ready to ship, or will be brought back<!-- todo: clarify "brought back" - removed from OT portal? --> for further development based on developer feedback. 
```

Comment:
Clarify - "brought back" means that the feature is being removed from the OT portal and is not being added to the next versions of the browser.  
need wording to contrast w/ "ready to ship"
@leahmsft

Reply:
in 3288, suggested changing to: 
 or the feature will be withheld for further development